src/test/indexTest.js

```diff
@@ -9,10 +9,11 @@ const should = chai.should();
 
 chai.use(chaiHttp);
 
-describe('test', () => {
-  it('get', (done) => {
-    chai.request(app).get('/').end((err, res) => {
+describe('GET weather', () => {
```

Comment:
Façam mais testes, vocês só estão testando o fluxo principal, mas como vocês podem garantir que quando der erro, vai dar o erro que vocês esperam dar? Dois fluxos importantes: enviar algo diferente do que aceita (pelo que vi só aceita long) e se mandar um long que não representa nenhum local. Pensem em mais fluxos.

Reply:
A respeito desses testes, a API já retorna um erro dela em JSON caso valores inválidos sejam inseridos - sejam eles de locais inválidos ou tipos inválidos - então está querendo dizer que devemos testar valores inválidos para ver se ela vai realmente retornar esse erro?

E não compreendi muito bem a respeito de só aceitar long, testamos e me recordo de aceitar valores do tipo float, que é o que a API aceita.

Reply:
Em relação a primeira pergunta: é isso mesmo, mesmo o node já fazendo isso por default, precisamos testar para saber se está vindo como o esperado.

Sobre a segunda pergunta: é float mesmo, eu que me confundi na hora. Faz um teste colocando um string ou algo assim.

PS: aceitar string não é melhor não?

Reply:
Bem, a API só aceita float, e por consequência int, então não teria como aceitar string, acredito eu.
Inserindo 'http://localhost:3000/request/?lati=2.31&long=afafas' por exemplo retorna:
`{"cod":"400","message":"afafas is not a float"}`

Embora, para a API, não creio que há diferença as variáveis que recebem os valores de query no código serem 2.66 ou '2.66'.

Reply:
Tudo bem, pensei que era algo que estávamos tratando na nossa API. Mas do mesmo jeito faça os testes